### DISCUSSION TITLE
Suggested order of reading the tutorials.

### DISCUSSION THREAD

**Discussion by zemega** [jakobrunge/tigramite#228] [Q&A]
Hello.

I am new to the subject of causality in earth science. I was introduced to this analysis by RMetS Masterclass "Extracting Casual Information from Climate Data" on March 2022. I'm reading through the tutorials currently. I could not find any continuity between the tutorials pages.

Would it be possible to get a suggested order of reading for the tutorials. I have read the publications related and still re-reading them now.

**Comment by jakobrunge** [comment]
Start with the overview: https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_causal_discovery_overview.ipynb

Or read the paper first: https://github.com/jakobrunge/tigramite/blob/master/tutorials/Runge_Causal_Inference_for_Time_Series_NREE.pdf
